src/components/FloatingDropdown/FloatingDropdown.tsx

```diff
@@ -28,18 +31,24 @@ export interface FloatingDropdownProps {
   value?: string;
   /** Force dropdown open. Use for visual test. Optional. Boolean. */
   forceContentElementVisibility?: boolean;
+  /** Placement for Popover. Optional. Placement. */
+  placement?: Placement;
```

Comment:
Why was this prop added?

Reply:
Because of typescript changes, we had to send a placement prop to the core dropdown. I thought it should be passed like the rest of the props and not to be permanent.

Reply:
I see.
Too bad the wix-ui-core Dropdown doesn't have a default `auto` placement.
Should our default be `auto` instead of `bottom` 🤔  or is `bottom` the behaviour we're expecting?